pulsar-io/kafka-connect-adaptor/src/test/java/org/apache/pulsar/io/kafka/connect/KafkaConnectSourceTest.java

```diff
@@ -37,6 +36,7 @@
 import org.apache.pulsar.common.schema.KeyValue;
 import org.apache.pulsar.functions.api.Record;
 import org.apache.pulsar.io.core.SourceContext;
+import org.jetbrains.annotations.NotNull;
```

Comment:
This was added by the IDE.
We should remove it before committing

Reply:
Fixed